### PR TITLE
Add telemetry no update exception

### PR DESCRIPTION
**Pull request by oysand**

### CODE REVIEW

File: src/robot_interface/models/exceptions/robot_exceptions.py [hunk 1]
```diff
@@ -202,6 +202,17 @@ def __init__(self, error_description: str) -> None:
     pass
 
 
+# An exception which should be thrown by the robot package if there is no new telemetry update.
+class RobotTelemetryNoUpdateException(RobotException):
+    def __init__(self, error_description: str) -> None:
+        super().__init__(
+            error_reason=ErrorReason.RobotTelemetryPoseException,
```
Comment (Christdej) [inline src/robot_interface/models/exceptions/robot_exceptions.py:209]: Not fitting ErrorReason?

File: src/robot_interface/models/exceptions/robot_exceptions.py [hunk 2]
```diff
@@ -202,6 +202,17 @@ def __init__(self, error_description: str) -> None:
     pass
 
 
+# An exception which should be thrown by the robot package if there is no new telemetry update.
+class RobotTelemetryNoUpdateException(RobotException):
```
Comment (Christdej) [inline src/robot_interface/models/exceptions/robot_exceptions.py:206]:
```suggestion
class RobotTelemetryNoUpdateException(RobotTelemetryException):
```

File: src/robot_interface/telemetry/mqtt_client.py [hunk 3]
```diff
@@ -74,7 +75,7 @@ def run(self, isar_id: str, robot_name: str) -> None:
             try:
                 payload = self.telemetry_method(isar_id=isar_id, robot_name=robot_name)
                 topic = self.topic
-            except RobotTelemetryPoseException:
+            except (RobotTelemetryPoseException, RobotTelemetryNoUpdateException):
```
Comment (Christdej) [inline src/robot_interface/telemetry/mqtt_client.py:78]:
```suggestion
            except (RobotTelemetryException):
```

**Review by Christdej — CHANGES_REQUESTED** [review]

File: src/robot_interface/telemetry/mqtt_client.py (hunk 3 above)
Comment (oysand) [reply src/robot_interface/telemetry/mqtt_client.py:78]: The RobotTelemetryException is a more general exception that is caught below and sends a cloud health topic to flotilla

File: src/robot_interface/models/exceptions/robot_exceptions.py (hunk 2 above)
Comment (oysand) [reply src/robot_interface/models/exceptions/robot_exceptions.py]: It complains when adding that the RobotTelemetryException is trying to initianlize with its own error reason.
src/robot_interface/models/exceptions/robot_exceptions.py:185: note: "__init__" of "RobotTelemetryException" defined here
src/robot_interface/models/exceptions/robot_exceptions.py:209: error: Unexpected keyword argument "error_reason" for "__init__" of "RobotTelemetryException"  [call-arg]

Comment (Christdej) [reply src/robot_interface/models/exceptions/robot_exceptions.py:207]: I see

**Review by Christdej — APPROVED** [review]

**Pull request merged**